index.js

```diff
@@ -1,13 +1,64 @@
-const books = localStorage.getItem('books') !== null
-  ? JSON.parse(localStorage.getItem('books'))
-  : [];
-
 const title = document.getElementById('title');
 const author = document.getElementById('author');
 const addBtn = document.getElementById('add-btn');
 let inputtedAuthor = '';
 let inputtedTitle = '';
 
+class Book {
```

Comment:
- [ ] You seem to have dumpled styles directly in the `style tag` in `index.html`, kindly remove it and place it in a stylesheet. It makes the codebase neater.